### ISSUE TITLE
Refactor styles for mobile display

### ISSUE THREAD

**Issue by queicherius** [gw2efficiency/issues#68]
I want to extend the frontend so it feels like a native application on mobile devices. This requires quite a lot of stuff to be done before, so I don't have to do everything twice. At that point I will probably split this into individual issues.

---

Prerequisites: #22

**Comment by queicherius** [comment]
After reconsideration, this is not a feature that is worth the time investment for the gained value in my opinion, so I am going to close this ticket.

**Comment by queicherius** [comment]
(More or less closed because it's not one ticket, but an ongoing process)
